### PR TITLE
[artifactory][artifactory-ha] Add condition to disable migration init container

### PR DESCRIPTION
**Pull request by ronnyn**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by chukka** [comment]
@ronnyn Can you please update changelog file . Also please add the same change for `artifactory` chart` in the same PR

File: stable/artifactory-ha/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [3.0.7] - Jul 23, 2020
```
Comment (chukka) [inline stable/artifactory-ha/CHANGELOG.md:4]: @ronnyn Please bump version here

**Comment by chukka** [comment]
@ronnyn Can you add changelog and chart.yaml  changes in `artifactory`

File: stable/artifactory/templates/artifactory-statefulset.yaml [hunk 2]
```diff
@@ -217,6 +217,7 @@ spec:
     {{- if .Values.artifactory.customInitContainers }}
 {{ tpl .Values.artifactory.customInitContainers . | indent 6 }}
     {{- end }}
+{{- if .Values.artifactory.migration.enabled }}
```
Comment (danielezer) [inline stable/artifactory/templates/artifactory-statefulset.yaml:220]: @ronnyn Please add `migration.enabled` to the artifactory `values.yaml` as well

**Pull request merged**